### PR TITLE
[FIX] ignore exceptions when moving attachment

### PR DESCRIPTION
**Pull request by hbrunn**

### CODE REVIEW

File: attachments_to_filesystem/models/ir_attachment.py [hunk 1]
```diff
@@ -102,12 +102,15 @@ def _attachments_to_filesystem_cron(self, cr, uid, context=None,
...
+            try:
+                ir_attachment.write(
+                    cr, uid, [attachment_id],
+                    {
+                        'datas': attachment_data['datas'],
+                        'db_datas': False,
+                    },
+                    context=context)
+            except Exception as e:
+                logging.exception(e)
```
Comment (pedrobaeza) [inline attachments_to_filesystem/models/ir_attachment.py]: Better to use parameter exc_info for dumping full stack.

**Review by pedrobaeza — APPROVED** [review]
The Travis is error is unrelated to this module, but it would be good to be repaired in any moment (it's document_sftp module).

For this one, a little suggestion.

**Comment by hbrunn** [comment]
@pedrobaeza thanks, travis is fixed in https://github.com/OCA/knowledge/pull/130

**Comment by pedrobaeza** [comment]
Please rebase

**Comment by hbrunn** [comment]
thanks. The same as making https://github.com/OCA/knowledge/pull/128 green

**Comment by pedrobaeza** [comment]
Fast-tracked as these are linter PRs.

**Pull request merged**